src/Raven.Server/Rachis/RachisConsensus.cs

```diff
@@ -1112,6 +1116,13 @@ public void AcceptNewConnection(Stream stream, Action disconnect, EndPoint remot
             }
         }
 
+        private static void ValidateCompatibility(RachisHello initialMessage)
+        {
+            var version = initialMessage.ServerBuildVersion;
+            if (version == 0 || ServerVersion.IsNightlyOrDev(version))  
+                return;
+        }
```

Comment:
What is the point of this? Preparing for the future?

Reply:
Yes, correct. In theory we can remove this, but maybe it is better to have this extension point prepared.